backend/Testing/Services/JwtHelper.cs

```diff
@@ -0,0 +1,28 @@
+using System.Net.Http.Json;
+using LexBoxApi.Auth;
+using Shouldly;
+
+namespace Testing.Services;
+
+public class JwtHelper
+{
+
+    public static async Task<string> GetJwtForUser(SendReceiveAuth auth)
+    {
+        var handler = new HttpClientHandler();
+        var client = new HttpClient(handler);
```

Comment:
we need to avoid creating a new http client like this. Sadly dotnet does not handle this very well and you can exhaust connections this way. I'd create a static client in this class and use that, just make sure to clear cookies on each call so the state doesn't leak between calls.

[Official Recommendation](https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines#recommended-use), yes this is poorly designed.